sycl/include/CL/sycl/detail/defines.hpp

```diff
@@ -18,6 +18,10 @@
 #define __has_attribute(x) 0
 #endif
 
+#ifndef __has_builtin
```

Comment:
Just for sake of readability, could you move this block above `__has_attribute` check?

Reply:
Moved it below instead to make stuff more alphabetical. Would this work as well?

Reply:
Sure. Thanks.